changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+#### Unreleased changes
```

Comment:
Remember to update release-date

Reply:
I have more changes coming... 😏

Reply:
So this change is to be bundled with other PRs? I guess that could work, we'd just have to schedule the PRs to happen somewhat after one another and the changelog to contain all PRs changes.

This smells like it could use a `dev` branch though.